toucan_connectors/google_credentials.py

```diff
@@ -11,16 +12,17 @@ class GoogleCredentials(BaseModel):
     type: str = Field(
         'service_account', title='Service account', description=CREDENTIALS_INFO_MESSAGE
     )
-    project_id: str = Field(..., title='Project ID', description=CREDENTIALS_INFO_MESSAGE)
-    private_key_id: str = Field(..., title='Private Key ID', description=CREDENTIALS_INFO_MESSAGE)
+    # On service account MODE :
+    project_id: str = Field(None, title='Project ID', description=CREDENTIALS_INFO_MESSAGE)
```

Comment:
were the fields nullable before introducing the `...` to `None` change?

Reply:
makes senses 👍🏽 
updated here : [`e4ee307` (#1287)](https://github.com/ToucanToco/toucan-connectors/pull/1287/commits/e4ee307bf96a192a65910aac27137c3e5b7f0183)